### PR TITLE
Workflow / process usage

### PR DESCRIPTION
**Pull request by netskink**
These are starter files for workflow description

### CODE REVIEW

File: setenv.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# update the path
+export PATH=${HOME}/zopen/dev/depot_toolsport/depot_tools:$PATH
+
+# activate the virtual env
+. ./myenv/bin/activate
+
+# Disable updating depot_tools
+export DEPOT_TOOLS_UPDATE=0
```
Comment (gabylb) [inline setenv.sh:10]: In my env I also have:
```
export DEPOT_TOOLS_BOOTSTRAP_PYTHON3=0
```
so `bootstrap_python3` doesn't run (by `ensure_bootstrap`; it's also run by `update_depot_tools` but the above envar already prevents it from running).

File: setenv.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Using depot_tools disable metrics.  
+# Comment out this line if you wish to participate
+gclient metrics --opt-out
+
+
+
```
Comment (gabylb) [inline setenv.sh]: Are those 3 blank lines needed?

File: setenv.sh (hunk 1 above)
Comment (netskink) [reply setenv.sh:10]: Thanks for the update.  FWIW, I also removed the trailing spaces after one of the lines.  I could care less about such things, but I need to get back into the habit before we attempt to upstream.

File: setenv.sh (hunk 2 above)
Comment (netskink) [reply setenv.sh]: See above.

**Review by gabylb — APPROVED** [review]

**Comment by netskink** [comment]
Hello Gaby,

Thanks for approving this.  I appreciate your time and expertise.

John

**Pull request merged**